flexget/plugins/clients/rtorrent.py

```diff
@@ -387,6 +425,29 @@ def _build_options(self, config, entry, entry_first=True):
                 elif entry_value:
                     options[opt_key] = entry.render(entry_value)
 
+        # Add custom fields on to options
+        entry_custom_fields = entry.get('custom_fields')
+        config_custom_fields = config.get('custom_fields')
+
+        if entry_custom_fields is None:
+            entry_custom_fields = {}
+
+        if config_custom_fields is None:
+            config_custom_fields = {}
+
+        merged_custom_fields = {}
+        if entry_first:
+            merged_custom_fields.update(entry_custom_fields)
+            merged_custom_fields.update(config_custom_fields)
+        else:
+            merged_custom_fields.update(config_custom_fields)
+            merged_custom_fields.update(entry_custom_fields)
+
+        if len(merged_custom_fields) > 0:
```

Comment:
The normal way to do a check like this in python is like this:
```suggestion
        if merged_custom_fields:
```